tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/HealthChecks/HealthChecksSteps.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.API.IntegrationTests.Steps.Common;
+using NHSD.BuyingCatalogue.API.IntegrationTests.Support;
+using NHSD.BuyingCatalogue.Testing.Data;
+using TechTalk.SpecFlow;
+using WireMock.Admin.Mappings;
+
+
+namespace NHSD.BuyingCatalogue.API.IntegrationTests.Steps.HealthChecks
+{
+    [Binding]
+    internal sealed class HealthChecksSteps
+    {
+        private readonly Response _response;
+        private readonly ScenarioContext _context;
+        private const string BaseUrl = "http://localhost:8080";
+        private const string ScenarioContextMappingKey = "HealthCheckMappingGuids";
+
+        public HealthChecksSteps(Response response, ScenarioContext context)
+        {
+            _response = response;
+            _context = context;
+        }
+
+        [Given(@"The DocumentAPI is (up|down)")]
+        public async Task GivenTheDocumentApiIs(string state)
```

Comment:
Should probably say the document api liveness healthcheck....

Reply:
As per our discussion: This test already exists, before any tests are run, the liveness of bapi is asserted [here](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueService/blob/development/tests/NHSD.BuyingCatalogue.API.IntegrationTests/Support/DocumentService.cs)